kernel/arch/dreamcast/kernel/init.c

```diff
@@ -71,9 +70,14 @@ void arch_init_net_dcload_ip(void) {
                ip.ipb[2], ip.ipb[1], ip.ipb[0]);
         dbgio_disable();
     }
-
-    net_init(ip.ipl);     /* Enable networking (and drivers) */
-
+    
+    /* Enable networking (and drivers) */
+    if((__kos_init_flags & INIT_NET_STATIC) == INIT_NET_STATIC)
```

Comment:
```suggestion
    if(__kos_init_flags & INIT_NET_STATIC)
```
When we only have one bit in a mask we're testing against (ie a bitflat), we just check it like that, since *any* nonzero value implies equality here. This is also consistent with the rest of the coedz.

Reply:
INIT_NET_STATIC is the ORed value of 0x100 and INIT_NET.  So would this give a false positive if INIT_NET is used instead of INIT_NET_STATIC?